include/velocypack/Options.h

```diff
@@ -116,6 +116,9 @@ struct Options {
   // JSON with a Dumper
   bool escapeForwardSlashes = false;
 
+  // with a Dumper (creates \uxxxx sequences or displays '\n', '\r' or \'t')
```

Comment:
The comment here does not really indicate what happens if you turn the option off.
If the option is set to false, all unicode and control characters are _suppressed_ (or replaced by whitespace).
So it would be better to rename to option to `suppressControlChars`. But actually it doesn't only suppress control chars, but also other characters, such as `/`, `"`, `\`. I don't think this is the intended behavior.